nomad/structs/workload_id.go

```diff
@@ -302,6 +302,7 @@ type WIHandle struct {
 	IdentityName string
 	// WorkloadIdentifier is either a ServiceName or a TaskName
 	WorkloadIdentifier string
+	VaultRole          string
```

Comment:
```suggestion
```

I don't think we need this anymore?

Reply:
Oh damn, sorry I missed this 🤦- thought this was the Claims struct when I was self-reviewing